### PR TITLE
Match designs for mobile chat input

### PR DESCRIPTION
**Pull request by siepra**
### Pull Request Checklist

- [ ] I have linked this PR to related GitHub issue.
- [ ] I have updated the CHANGELOG.md file with relevant changes (the file is located at the root of monorepo).

### CODE REVIEW

File: packages/mobile/src/components/Chat/Chat.stories.tsx [hunk 1]
```diff
@@ -221,20 +221,6 @@ storiesOf('Chat', module)
                 pubKey: 'test',
               },
             ],
-            [
```
Comment (EmiM) [inline packages/mobile/src/components/Chat/Chat.stories.tsx:224]: Why did you remove this long message from stories?

Comment (siepra) [reply packages/mobile/src/components/Chat/Chat.stories.tsx:224]: Because it brings no value and makes the story messy

Comment (vinkabuki) [reply packages/mobile/src/components/Chat/Chat.stories.tsx]: could you add a comment explaining this? It's indeed misleading and tempting to remove unless you know the whole story behind it - it's testing proper line breaking

**Review by vinkabuki — APPROVED** [review]
